pkg/clustermanager/eksa_installer.go

```diff
@@ -108,6 +109,47 @@ func (i *EKSAInstaller) Upgrade(ctx context.Context, log logr.Logger, c *types.C
 	return changeDiff, nil
 }
 
+// ApplyBundles applies the bundles to the cluster.
+func (i *EKSAInstaller) ApplyBundles(ctx context.Context, log logr.Logger, cluster *types.Cluster, spec *cluster.Spec) error {
```

Comment:
Can all these methods be part of `Install`? (you can make them unexported and then call them from `Install`)